/crazyflie_server:
  ros__parameters:
    # simulation related
    sim:
      max_dt: 0 #0.1              # artificially limit the step() function (set to 0 to disable)
      backend: np                 # see backend folder for a list 
      visualizations:             # see visualization folder for a list
        rviz:
          enabled: true
        pdf:
          enabled: false
          output_file: "result.pdf"
        record_states:
          enabled: false
          output_dir: "state_info"
          logging_time: 0.2  # how many seconds to leave between logs
          file_formats: ["csv", "np"] # csv, np
      controller: mellinger # none, pid, mellinger
